src/urbit/app/chess.hoon

```diff
@@ -25,7 +25,7 @@
 +$  state-1
   $:  %1
       games=(map @dau active-game-state)
-      archive=(map @dau chess-game)
+      archive=((mop @dau chess-game) lth)
```

Comment:
do we think we'll need to make anything else in state a `mop`?

Reply:
See [here](https://github.com/thecommons-urbit/chess/pull/109#discussion_r1095046515).

Reply:
Go [there](https://github.com/thecommons-urbit/chess/pull/109#discussion_r1095046515).

Reply:
Should be resolved by [this comment](https://github.com/thecommons-urbit/chess/issues/65#issuecomment-1424056224) on #65; please reopen conversation if that's not the case.